buildSrc/src/main/kotlin/version.gradle.kts

```diff
@@ -0,0 +1,65 @@
+import org.eclipse.kuksa.util.VersionProperties
+
+val properties = VersionProperties("$rootDir/version.properties")
+properties.load()
+
+rootProject.extra["projectVersion"] = properties.version
+rootProject.extra["projectVersionCode"] = properties.versionCode
+
+tasks.register("increaseMajorVersion") {
+    group = "version"
+    doLast {
+        properties.major += 1
```

Comment:
reset minor and patch to 0 when updating major version?

Reply:
Yes please :)